mono/mini/debugger-agent.c

```diff
@@ -594,7 +605,7 @@ typedef struct {
 	char *category, *message;
 	/* For EVENT_KIND_TYPE_LOAD */
 	MonoClass *klass;
-} EventInfo;
+} DebuggerEventInfo;
```

Comment:
Did you rename this because Il2CPP has a struct of the same name? Seems to me that this could be a source of merge pains and we should renamie the IL2CPP struct instead.

Reply:
Yes, we needed to avoid conflicting with `EventInfo` in the libil2cpp code. I'm not sure which is better to change. I'm fine with changing either one. I think I searched and found this was used less often, so I changed it.

We could also consider upstreaming this change to prevent merge issues. I'm on the fence about what to do here.